components/date-picker/src/DatePicker.svelte

```diff
@@ -168,7 +173,7 @@
         bind:month
         bind:year
         {disabledDate}
-        on:change={handleSelectDate}
+        on:change={(v) => setDateOnlyIfValid(v.detail)}
```

Comment:
please help to rename `setDateOnlyIfValid` to `handleCalendar` and rename `handleChange` to `handleInput`